src/main/model/ProxyCommand.java

```diff
@@ -0,0 +1,37 @@
+package main.model;
+
+public class ProxyCommand {
+    private String url;
```

Comment:
Again, might want a unique name for this - perhaps ``proxyServerUrl`` or similar?